appintro/src/main/java/com/github/appintro/indicator/DotIndicatorController.kt

```diff
@@ -67,7 +70,12 @@ class DotIndicatorController(context: Context) : IndicatorController, LinearLayo
             } else {
                 unselectedIndicatorColor
             }
-            (getChildAt(i) as ImageView).let { DrawableCompat.setTint(it.drawable, tint) }
+            (getChildAt(i) as ImageView).let {
+                DrawableCompat.setTint(it.drawable, tint)
+            }
+            if (slideCount == 1) {
+                (getChildAt(i) as ImageView).visibility = View.INVISIBLE
+            }
```

Comment:
```suggestion
            (getChildAt(i) as ImageView).let { DrawableCompat.setTint(it.drawable, tint) }
```

Those changes are unnecessary and can probably be removed.